docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/rdb-loader-configuration-reference/index.md

```diff
@@ -27,11 +27,14 @@ This is a complete list of the options that can be configured:
 | `host`                     | Required. Host name of Redshift cluster.                                                                                                                                        |
 | `port`                     | Required. Port of Redshift cluster.                                                                                                                                             |
 | `database`                 | Required. Redshift database which the data will be loaded to.                                                                                                                   |
-| `roleArn`                  | Required. AWS Role ARN allowing Redshift to load data from S3.                                                                                                                  |
+| `roleArn`                  | Required if 'NoCreds' is chosen as load auth method.. AWS Role ARN allowing Redshift to load data from S3.                                                                                                                  |
 | `schema`                   | Required. Redshift schema name, eg “atomic”.                                                                                                                                    |
 | `username`                 | Required. DB user with permissions to load data.                                                                                                                                |
 | `password`                 | Required. Password of DB user.                                                                                                                                                  |
 | `maxError`                 | Optional. Configures the [Redshift MAXERROR load option](https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-data-load.html#copy-maxerror). The default is 10.       |
+| `loadAuthMethod.*`         | Optional, default method is `NoCreds`. Specifies the auth method to use with 'COPY' statement. |
+| `loadAuthMethod.type`      | Required if `loadAuthMethod` section is included. Specifies the type of the auth method. The possible values are `NoCreds` and `TempCreds`. <br/><br/>With 'NoCreds', no credentials will be passed to 'COPY' statement. Instead, Redshift cluster needs to be configured AWS Role ARN allowing Redshift to load data from S3 and this Role ARN needs to be passed field 'roleArn' above. More information can be found [here](https://docs.aws.amazon.com/redshift/latest/dg/copy-usage_notes-access-permissions.html). <br/><br/>With 'TempCreds', temporary credentials will be created for every load operation and these temporary credentials will be passed to 'COPY' statement. |
```

Comment:
```suggestion
| `loadAuthMethod.type`      | Required if `loadAuthMethod` section is included. Specifies the type of the authentication method. The possible values are `NoCreds` and `TempCreds`. <br/><br/>With `NoCreds`, no credentials will be passed to the `COPY` statement. Instead, Redshift cluster needs to be configured with an AWS Role ARN that allows it to load data from S3. This Role ARN needs to be passed in the `roleArn` setting above. You can find more information [here](https://docs.aws.amazon.com/redshift/latest/dg/copy-usage_notes-access-permissions.html). <br/><br/>With `TempCreds`, temporary credentials will be created for every load operation and these temporary credentials will be passed to the `COPY` statement. |
```
Can we add any info on which method is recommended and why?

Reply:
Since we operating customer's pipelines and we don't have access to their Redshift instances directly, `TempCreds` method has some advantage for us since it has one less step to configure on customer side. However, for ordinary users, I don't think any method significant advantage over the other one. Therefore, I think there is no need to recommend any method.